docassemble/HousingCodeChecklist/data/questions/housing_code_interview.yml

```diff
@@ -54,6 +54,10 @@ default screen parts:
     <img src="${ uptocode_logo.url_for() }"><div class="title"></div>
   short logo: |
     <img src="${ uptocode_logo.url_for() }"><div class="title"></div>
+  navigation bar html: |
+    % if defined('acknowledged_information_use'):
+    ${ get_language_list(get_tuples(['en','es','pt','ht']),current=user_language) }
```

Comment:
I don't see the new `language.py` in this PR? Need to add to the package maybe?

Reply:
will do that now